### DISCUSSION TITLE
Is recruiting allowed on GitHub?

### DISCUSSION THREAD

**Discussion by shadow9900** [community/community#22294] [New to GitHub]
<p>Hi,</p>
<p>I’ve read over the code of conduct &amp; terms of service and didn’t see anything specifically against it, but is it considered non-pc here to recruit? I’m obviously new, so I figure I’ll settle in before I start going on a rule-breaking rampage.</p>
<p>If it’s allowed &amp; not frowned upon, would we ask in the Marketplace? or in the community?</p>
<p>(Sorry if this has been covered. I searched for both “recruit” and the main skill we’re looking for &amp; couldn’t find anything. )</p>
<p>Thanks for your help,</p>

**Comment by nadiajoyce** [comment, marked as answer]
<p>Hi <a class="mention" href="/shadow9900">@shadow9900</a>, we’re glad you’re here :grinning:</p>
<p>We don’t want to have job listings on the Community Forum right now. It is something that we’ve added to the list to investigate if we can do so without overwhelming other conversations. That said, if you have a project that you’re working on in a public repo on GitHub and you simply want to invite others to be outside collaborators and help with the project, it’s completely all right to share a link to your repo in the forum asking for specific feedback or help. </p>
<p>Keep checking back as the Community Forum grows and we add more capabilities!</p>
<p>Thanks for your interest :+1:</p>

**Comment by shadow9900** [comment]
<p>Thank you for your answer!</p>

**Comment by daniellampinen** [comment]
<p>In which forum category is it allowed to do this (invite people to be outside collaborators)?</p>

**Comment by ThirdEyeRose** [comment]
<p>Hi <a class="mention" href="/daniellampinen">@daniellampinen</a>,</p>
<p>If you’re looking for collaborators, you might want to check out the <a href="/t5/Programming-Help-and-Discussion/bd-p/programming">Programming Help and Discussion board</a>. Please make sure to <a href="https://github.community/t/welcome-to-the-programming-help-and-discussion-board/15036">read the welcome post</a> as it will help you to get better responses.</p>
<p>Thanks!</p>
